spec/classes/init_spec.rb

```diff
@@ -1,47 +1,53 @@
 require 'spec_helper'
 
 describe 'incron' do
-  context 'supported operating systems' do
-    on_supported_os.each do |os, facts|
-      context "on #{os}" do
-        let(:facts) do
-          facts
-        end
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      let(:facts) do
+        facts
+      end
 
-        context 'with default parameters' do
-          it { is_expected.to compile.with_all_deps }
-          it { is_expected.to create_class('incron') }
-          it { is_expected.to create_incron__user('root') }
-          it { is_expected.to create_concat('/etc/incron.allow') }
-          it { is_expected.to create_file('/etc/incron.deny').with({:ensure => 'absent'}) }
-          it { is_expected.to create_package('incron').with({:ensure => 'latest'}) }
-          it { is_expected.to create_service('incrond').with({
-            :ensure     => 'running',
-            :enable     => true,
-            :hasstatus  => true,
-            :hasrestart => true
-          }) }
-        end
+      context 'with default parameters' do
+        it { is_expected.to compile.with_all_deps }
+        it { is_expected.to create_class('incron') }
+        it { is_expected.to create_incron__user('root') }
+        it { is_expected.to create_concat('/etc/incron.allow') }
+        it { is_expected.to create_file('/etc/incron.deny').with({:ensure => 'absent'}) }
+        it { is_expected.to create_package('incron').with({:ensure => 'latest'}) }
+        it { is_expected.to create_service('incrond').with({
+          ensure:     'running',
```

Comment:
The use of the newer notation for Hashes is IMHO a style difference.  In the absence of coding standards, I generally try not to make style changes for the sake of making style changes. I also try to use the predominant style in a file.  This prevents style wars and makes code reviews easier.

Reply:
Agree with @lnemsick-simp 

I've been avoiding the new syntax because it can **only be used with symbols** which means you'll have a slew of old and new syntax across the file and I would like to suggest this as a global practice for the SIMP project for ease of consistent visual processing.